package.json

```diff
@@ -8,34 +8,44 @@
 		"yarn": ">=3.1.0"
 	},
 	"scripts": {
-		"dev": "nuxi dev",
-		"build": "nuxi build",
+		"dev": "vue-cli-service serve",
+		"build": "vue-cli-service build",
+		"lint": "vue-cli-service lint",
 		"start": "node .output/server/index.mjs",
 		"lint:js": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
 		"lint:prettier": "prettier --check .",
-		"lint": "yarn lint:js && yarn lint:prettier",
 		"lint:fix": "prettier --write --list-different . && yarn lint:js --fix",
-		"typecheck": "nuxi typecheck",
 		"test": "jest"
 	},
-	"devDependencies": {
-		"@nuxtjs/eslint-config-typescript": "^8.0.0",
-		"eslint": "^8.10.0",
-		"eslint-config-prettier": "^8.4.0",
-		"husky": "^7.0.4",
-		"jest": "^27.5.1",
-		"lint-staged": "^12.3.4",
-		"nuxi": "^0.10.1",
-		"nuxt3": "latest",
-		"prettier": "^2.5.1",
-		"typescript": "^4.6.2",
-		"vue-jest": "^3.0.7"
+	"dependencies": {
+		"@algo-builder/algob": "^3.2.0",
+		"@algo-builder/web": "^3.2.0",
+		"@ant-design/icons-vue": "^6.0.1",
+		"algosdk": "^1.10.0",
+		"ant-design-vue": "2.2.1",
+		"core-js": "^3.6.5",
+		"js-base64": "^3.6.1",
+		"vue": "^3.0.0",
+		"vue-router": "^4.0.10",
+		"vuex": "^4.0.2"
 	},
 	"lint-staged": {
 		"*.{js,ts,vue}": "eslint --cache",
 		"*.**": "prettier --ignore-unknown --write"
 	},
-	"dependencies": {
-		"ant-design-vue": "next"
+	"devDependencies": {
+		"@typescript-eslint/eslint-plugin": "^4.18.0",
+		"@typescript-eslint/parser": "^4.18.0",
+		"@vue/cli-plugin-babel": "~4.5.0",
+		"@vue/cli-plugin-eslint": "~4.5.0",
+		"@vue/cli-plugin-typescript": "~4.5.0",
+		"@vue/cli-service": "~4.5.0",
+		"@vue/compiler-sfc": "^3.0.0",
+		"@vue/eslint-config-typescript": "^7.0.0",
+		"eslint": "^6.7.2",
+		"eslint-plugin-vue": "^7.0.0",
+		"lint-staged": "^12.3.4",
+		"prettier": "^2.5.1",
+		"typescript": "~4.3.5"
```

Comment:
why using outdated typescript and eslint?

please run `yarn up` and make sure we are using the latest releases.

Reply:
ping